include/ginkgo/core/base/version.hpp

```diff
@@ -76,21 +76,17 @@ struct version {
 };
 
 
-#define GKO_ENABLE_VERSION_COMPARISON(_operator)                          \
-    bool operator _operator(const version &first, const version &second)  \
```

Comment:
I think we should leave this in the header and just add `inline`. Performance reasons, if someone goes crazy with tons of version comparisons.

Reply:
Makes sense, is now pushed.